test/SILGen/moveonly.swift

```diff
@@ -71,12 +73,12 @@ public func consumeVal(_ s: __owned NonTrivialStruct2) {}
 // Function Arguments
 //
 
-// CHECK-LABEL: sil [ossa] @$s8moveonly19useNonTrivialStructyyAA0cdE0VhF : $@convention(thin) (@guaranteed NonTrivialStruct) -> () {
+// CHECK-LABEL: sil [ossa] @$s8moveonly19useNonTrivialStructyyAA0cdE0VF : $@convention(thin) (@guaranteed NonTrivialStruct) -> () {
 // CHECK: bb0([[ARG:%.*]] : @guaranteed $NonTrivialStruct):
 // CHECK:   [[COPIED_ARG:%.*]] = copy_value [[ARG]]
 // CHECK:   mark_must_check [no_consume_or_assign] [[COPIED_ARG]]
-// CHECK: } // end sil function '$s8moveonly19useNonTrivialStructyyAA0cdE0VhF'
-public func useNonTrivialStruct(_ s: __shared NonTrivialStruct) {
+// CHECK: } // end sil function '$s8moveonly19useNonTrivialStructyyAA0cdE0VF'
+public func useNonTrivialStruct(_ s: borrowing NonTrivialStruct) {
```

Comment:
This mangling change surprises me.  I can see why `__shared` and `borrowing` might mangle differently, but we're just losing a character here, which looks like we're not including `borrowing` in the mangling at all?

Reply:
CC: @jckarter on the mangling question.  What do we expect to tell people who use `__shared` currently?   If the mangling is affected, then they can't replace that with `borrowing`.  Do we expect to support `__shared` indefinitely?

Reply:
@tbkka we should still ask @jckarter but since he is on vacation, I will give you my recollection. My recollection is that borrowing only affects the mangling if it changes the default convention. But my memory may be incorrect.